ci-operator/config/netobserv/network-observability-operator/netobserv-network-observability-operator-main.yaml

```diff
@@ -60,9 +60,7 @@ tests:
 - as: e2e-operator
   optional: true
   steps:
-    cluster_profile: aws-qe
-    env:
-      BASE_DOMAIN: qe.devcluster.openshift.com
+    cluster_profile: aws
```

Comment:
@jianlinliu what AWS accounts does this cluster_profile use? Any reason for this switch from QE account ?

Reply:
@memodi `aws` account is managed by DPTP team. Generally any installation does not rely on cluster profile, recently QE's aws account often hit quota limit, we need to balance the budget, and `e2e-operator` sounds not for QE specific testing, so switch the account.